portals/admin/src/main/java/org/wso2/carbon/apimgt/ui/admin/Util.java

```diff
@@ -0,0 +1,121 @@
+/*
+
+
+ *  Copyright (c) 2017, WSO2 LLC (http://www.wso2.org) All Rights Reserved.
+ * 
+ *  WSO2 LLC licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ * 
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+
+
+ */
+package org.wso2.carbon.apimgt.ui.admin;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Arrays;
+import java.util.Map;
+import javax.servlet.ServletContext;
+import org.wso2.carbon.apimgt.api.APIManagementException;
+import org.wso2.carbon.apimgt.impl.utils.APIUtil;
+
+public class Util {
+
+    /**
+     * Read a json file from the directory and output as a Map object.
+     *
+     * @param path    required parameter
+     * @param context required parameter
+     * @return Map<String, Object>
+     * @throws FileNotFoundException
+     */
+    public static Map<String, Object> readJsonFile(String path, ServletContext context) throws FileNotFoundException {
+        String realPath = context.getRealPath(path);
+        BufferedReader bufferedReader = new BufferedReader(new FileReader(realPath));
+        Gson gson = (new GsonBuilder()).setPrettyPrinting().create();
+        return (Map) gson.fromJson(bufferedReader, Map.class);
+    }
+
+    /**
+     * Returns the value in the given path of the nested tree map. <br> "." separate 2 levels in Map tree.
+     *
+     * @param json required parameter
+     * @param path required parameter
+     * @return Map<String, Object>
+     */
+    public static Object readJsonObj(Map json, String path) {
+        String[] pathStrings = path.split("\\.");
+        Map nestedJson = json;
+
+        for (String pathString : Arrays.copyOfRange(pathStrings, 0, pathStrings.length - 1)) {
+            if (!nestedJson.containsKey(pathString)) {
+                return null;
+            }
+            nestedJson = (Map) nestedJson.get(pathString);
+        }
+
+        if (!nestedJson.containsKey(pathStrings[pathStrings.length - 1])) {
+            return null;
+        }
+        return nestedJson.get(pathStrings[pathStrings.length - 1]);
+    }
+    
+    /**
+     * Get the loopback (localhost) origin (scheme + hostname + port), This origin is used for making
+     * internal(within the web app node), API calls.For example DCR call, Token generation, User Info, Token Introspect,
+ Revoke etc.
+     * @param host
+     * @return String
```

Comment:
Return type is already obvious from the code. Provide some description. (or remove the line completely if there is no useful description). Fix this in other places too in your code, considering other params too